### ISSUE TITLE
False positive "crate visibility modifier is experimental" for macro calls

### ISSUE THREAD

**Issue by dima74** [intellij-rust/intellij-rust#5492]
## Environment

* **IntelliJ Rust plugin version:** 0.2.123.3132-201
* **Rust toolchain version:** 1.43.1 (8d69840ab 2020-05-04) x86_64-unknown-linux-gnu
* **IDE name and version:** IntelliJ IDEA 2020.2 EAP Ultimate Edition (IU-202.4357.23)
* **Operating system:** Linux 5.6.15-arch1-1

## Problem description
![image](https://user-images.githubusercontent.com/6505554/83657946-fd56d000-a5da-11ea-91d2-b85ba77e6d53.png)

## Steps to reproduce
```rust
#[macro_export]
macro_rules! hello_macro {
    () => {
        println!("Hello!");
    };
}

fn main() {
    crate::hello_macro!();
}
```
